dashboard/app/views/home/index.html.haml

```diff
@@ -5,6 +5,10 @@
 
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
+- if current_user && current_user.show_race_interstitial?
+  - current_user.race_interstitial_shown = true
```

Comment:
The prerendering added by #11256 only affects the puzzle progression page (`levels/show.html.haml` view), while this view is only on the home page, so they shouldn't affect each other.

However, the hypothetical future case of either a prerender or some other speculative-fetching (for example, Chrome has a feature that will automatically prerender pages when the address is entered in the omnibox, regardless of any `prerender` link tag in the html) causing the interstitial to be shown but not actually viewed by the human user is worth considering.